project.clj

```diff
@@ -101,7 +101,7 @@
                        :dependencies [[puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]]}
              :ci {:plugins [[lein-pprint "1.1.1"]]}}
 
-  :test-selectors {:default (complement :integration)
+  :test-selectors {:default (constantly :all)
```

Comment:
Ohhh...judging from the `:integration :integration` selector, I bet when we try `:default :all` it's looking for tests annotated literally as "^:all", which would explain why 0 tests were ran.